pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+ 
+[project]
+name = "ts-observatory-control"
+description = "Observatory Control library."
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = { documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_observatory_control" }
+dynamic = [ "version" ]
+ 
+[tools.setuptools]
+package-data = { "" = "*.csv"}
+ 
+[tool.setuptools.dynamic]
+version = { attr = "setuptools_scm.get_version" }
+ 
+[tool.setuptools.packages.find]
```

Comment:
I noticed that none of the scripts in the `bin` directory are used for unit testing. Still, are you sure you don't want to include any `entry points` for them here?